lib/charms/prometheus_k8s/v0/prometheus.py

```diff
@@ -270,6 +316,8 @@ def _on_scrape_targets_changed(self, event):
 # to 0 if you are raising the major API version
 LIBPATCH = 2
 
+ALERT_RULES_PATH = "prometheus_alert_rules"
```

Comment:
I would prefix `./` for clarity

Reply:
Only reason I did not do that is so that this constant can be used as part of others for example in writing tests. Besides `./` can also give the misleading impression that it is relative to the `lib/charms/prometheus_k8s/v0` directory.